apps/sms/services/js/conversation/conversation_service.js

```diff
@@ -101,4 +165,9 @@ if (!('BridgeServiceMixin' in self)) {
       { methods: METHODS, streams: STREAMS, events: EVENTS }
     )
   );
+
+  // Automatically init service if it's run inside worker.
+  if ('importScripts' in self) {
```

Comment:
stackoverflow tells me `if (! self.document)` <= you're inside a worker. (maybe `if (!('document' in self))` works better for you ?). No DOM in a worker, and I don't think this will change soon.
I don't know if `self.window` exists in a worker ?